src/pages/sighting/AnnotationDetail.jsx

```diff
@@ -56,6 +60,14 @@ export default function AnnotationDetail({
     setNewTagSelectValue(null);
   };
 
+  const viewpoint = viewpointChoices.find(
+    viewpointChoice =>
+      get(annotation, 'viewpoint') === viewpointChoice.value,
```

Comment:
Can there not be more than one annotation with the same viewpoint in Codex?

Reply:
there could, but this would still work. we are iterating over the viewpointChoices from the constants file, which all have unique (and always present) `value` properties